pkg/trace/api/otlp_test.go

```diff
@@ -1274,6 +1397,138 @@ func trimSpaces(str string) string {
 	return out.String()
 }
 
+func makeSpanLinkSlice(traceId, spanId, traceState string, attrs map[string]string, dropped uint32) ptrace.SpanLinkSlice {
```

Comment:
nit: I usually think it's a good practice to test helper functions too. If it's a test helper, it doesn't mean it is reliable to work correctly 🙂

Reply:
I was on the fence about this one since, if there were any problems with it, then `TestMarshalSpanLinks` would likely fail but it certainly doesn't hurt to add some basic testing.

Done.